src/backend/PortabilityService.AnalysisEngine/appsettings.json

```diff
@@ -0,0 +1,3 @@
+﻿{
+  "BlobStorageConnectionString": "tbd"
```

Comment:
Could we leave this undefined in production? We have other, more flexible configuration sources there.

Reply:
Removed

---
In reply to: [186501307](https://github.com/Microsoft/dotnet-apiport/pull/635#discussion_r186501307) [](ancestors = 186501307)